src/paradigma/tremor/tremor_analysis.py

```diff
@@ -11,20 +11,61 @@
 
 from paradigma.constants import DataColumns
 from paradigma.config import TremorFeatureExtractionConfig, TremorDetectionConfig, TremorQuantificationConfig
-from paradigma.tremor.feature_extraction import extract_spectral_domain_features
-from paradigma.segmenting import tabulate_windows_legacy
+from paradigma.tremor.feature_extraction import extract_spectral_domain_features, extract_spectral_domain_features
+from paradigma.segmenting import tabulate_windows, tabulate_windows_legacy
```

Comment:
I think you can remove `tabulate_windows_legacy` if you switched to `tabulate_windows`, but we can move it to a next PR along with removing the function `tabulate_windows_legacy` altogether (after the PPG changes are merged).